projects/akita-ng-fire/src/lib/utils/types.ts

```diff
@@ -26,6 +26,7 @@ export interface WriteOptions extends PathParams {
 export interface SyncOptions extends PathParams {
   storeName: string;
   loading: boolean;
+  reset: boolean;
```

Comment:
Could you add a comment above this params to explain its purpose to developers ? (with JSDoc synthax `/** Will reset the store on .... */`)

Reply:
Done